testing/internal/e2e/scopes/scope_test.go

```diff
@@ -0,0 +1,49 @@
+package scope_test
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/boundary/testing/internal/e2e"
+	"github.com/hashicorp/boundary/testing/internal/e2e/boundary"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestListKeyVersionDestructionJobsCli uses the boundary CLI to list
+// pending key version destruction jobs.
+func TestListKeyVersionDestructionJobsCli(t *testing.T) {
```

Comment:
Is there a workflow that can be tested here? Generally I think these tests shouldn't just test a single cli command, you already added tests for that in the cli (bats) tests.

Reply:
There will be a workflow once we have the ability to trigger a key destruction. I could remove these tests until we can do them all together. I'm also not sure what the overlap between bats and these are, IMO we could replace the bats tests with this new framework.

Reply:
My understanding of the differences between the tests is that the bats tests are for testing the cli ui, so testing individual commands and getting expected outputs, while theses tests are for testing end-to-end admin workflows that will often involve multiple commands, and could be accomplished via the cli/api/admin ui.

I agree that the cli ui tests *could* be implemented using some of this code, instead of using bats, but I don't think there are any current plans to do so.